cluster/gce/gci/configure-helper.sh

```diff
@@ -725,16 +725,20 @@ function start-kube-addons {
     base_metrics_memory="140Mi"
     metrics_memory="${base_metrics_memory}"
     base_eventer_memory="190Mi"
+    base_metrics_cpu="80m"
+    metrics_cpu="${base_metrics_cpu}"
     eventer_memory="${base_eventer_memory}"
     nanny_memory="90Mi"
     local -r metrics_memory_per_node="4"
+    local -r metrics_cpu_per_node="0.5"
     local -r eventer_memory_per_node="500"
     local -r nanny_memory_per_node="200"
     if [[ -n "${NUM_NODES:-}" && "${NUM_NODES}" -ge 1 ]]; then
       num_kube_nodes="$((${NUM_NODES}+1))"
       metrics_memory="$((${num_kube_nodes} * ${metrics_memory_per_node} + 200))Mi"
       eventer_memory="$((${num_kube_nodes} * ${eventer_memory_per_node} + 200 * 1024))Ki"
       nanny_memory="$((${num_kube_nodes} * ${nanny_memory_per_node} + 90 * 1024))Ki"
+      metrics_cpu=$(echo - | awk "{print ${num_kube_nodes} * ${metrics_cpu_per_node} + 80}")m
```

Comment:
@zmerlynn - it's already 0.5 millicpu (0.0005 cpu). So if we want to keep it integer, we should probably move to micro-cpus...